src/services/trip.services.js

```diff
@@ -0,0 +1,115 @@
+import Db from "../database/models";
+
+const TripRequest = Db.TripRequest;
+
+export const getAllTripRequests = async (user) => {
+  let result;
+
+  if (user.user_role == "6927442b-84fb-4fc3-b799-11449fa62f52") {
+    result = await TripRequest.findAll({
```

Comment:
Instead of hardcoding this id, you can use findRoleById(roleId) service in roleService.js for all hardcode id in this tip.service file

Reply:
Ok will do